libs/components/forms/src/lib/modules/radio/radio.component.ts

```diff
@@ -215,66 +232,78 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
   @Output()
   // eslint-disable-next-line @angular-eslint/no-output-native
   public get change(): Observable<SkyRadioChange> {
-    return this._change;
+    return this.#_change;
   }
 
   /**
    * Fires when the selected value changes.
    */
   @Output()
   public get checkedChange(): Observable<boolean> {
-    return this._checkedChange;
+    return this.#checkedChangeObs;
   }
 
   /**
    * Fires when the selected value changes.
    */
   @Output()
   public get disabledChange(): Observable<boolean> {
```

Comment:
☝️ Just noticed this, but Angular recommends outputs be of type `EventEmitter`, but we're using `Observable`. Can you put `// TODO: convert to EventEmitter in a breaking change` above each one?

Reply:
Done